src/zappy_ai_src/ai_function_wrapper.py

```diff
@@ -86,9 +86,8 @@ def getNecessaryFunctions(self) -> bool:
             self.__GetRepPlaceObject = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_place_response")
             self.__GetRepIncantation = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_incantation_response")
             self.__GetNetworkState = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_network_state")
-            self.DLLibWrapper.closeLibray()
         except:
-            print("The provided lib doesn't contain all the required funtions.", file=stderr)
+            print("The provided lib doesn't contain all the required functions.", file=stderr)
```

Comment:
@31Nathan Altough this is descriptive, you really should consider giving more info
```suggestion
           except Execption e:
               printf(e) <-- Something like taht
               print("The provided lib doesn't contain all the required functions.", file=stderr)
```